Source/WTF/wtf/MemoryPressureHandler.cpp

```diff
@@ -85,12 +85,6 @@ void MemoryPressureHandler::setMemoryFootprintPollIntervalForTesting(Seconds pol
 
 void MemoryPressureHandler::setShouldUsePeriodicMemoryMonitor(bool use)
 {
-    if (!isFastMallocEnabled()) {
```

Comment:
What's the reason for this change? I think this will break malloc stack logging on Apple platforms.

Reply:
Well, it won't break it but it might make that mode more prone to memory kills.

Reply:
I moved this check to the caller (well, I guess only for the web content process) so that it can check JSC options too.

I can add it in for the other processes too if they might get frequently killed

Reply:
Ah I missed that, thanks.